stdlib/REPL/src/REPLCompletions.jl

```diff
@@ -970,8 +970,12 @@ function complete_keyword_argument(partial, last_idx, context_module)
     end
 
     suggestions = Completion[KeywordArgumentCompletion(kwarg) for kwarg in kwargs]
-    append!(suggestions, complete_symbol(nothing, last_word, Returns(true), context_module))
-    append!(suggestions, complete_keyval(last_word))
+
+    # Only add these if not in kwarg space. i.e. Not in `foo(; `
+    if !occursin(r"\([.\s\w,]*;[.\s\w,]+?$", partial)
```

Comment:
```suggestion
    # Only add these if not in kwarg space. i.e. not in `foo(; `
    if kwargs_flag == 0
```

The meaning of `kwargs_flag` is detailed above: https://github.com/JuliaLang/julia/blob/63299e5d6be0e9626f86a0335c0b3436db44b9c8/stdlib/REPL/src/REPLCompletions.jl#L743-L747 although I understand it's not easily discoverable. But it's always better to avoid having even one more regex!

Reply:
Great, yeah missed this. Thanks